action.js

```diff
@@ -275,24 +275,32 @@ async function findTaskInSection(client, sectionId, name) {
             } else {
                 console.info('task found', task.gid);
                 existingTaskId = task.gid
-            }            
+            }
         });
     } catch (error) {
         console.error('rejecting promise', error);
     }
     return existingTaskId
 }
 
-async function createTask(client, name, description, projectId, sectionId = '', tags = [], collaborators = []) {
+async function createTask(client, name, description, projectId, sectionId = '', tags = [], collaborators = [], assignee = '', customFields = '') {
     const taskOpts = {
-        name: name, 
-        notes: description, 
+        name: name,
+        notes: description,
         projects: [projectId],
         tags,
         followers: collaborators,
         pretty: true
     };
 
+    if (assignee != '') {
+        taskOpts.assignee = assignee;
+    }
+
+    if (customFields != '') {
+        taskOpts.custom_fields = JSON.parse(customFields);
```

Comment:
Err out if parsing fails?

Reply:
I have it catch and throw an error in console, not sure if want to hard-fail here in case someone accidentally deletes a custom field in a project?